aws/logs_monitoring/lambda_function.py

```diff
@@ -75,6 +75,15 @@ def __init__(self, name, pattern, placeholder):
     ),
 ]
 
+# DD_CUSTOM_SCRUBBING_RULE: Regular expression for custom scrubbing rule
+DD_CUSTOM_SCRUBBING_RULE = None
```

Comment:
You don't need to rewrite the logic of checking if the environment variable is set, you can just add a new entry to [SCRUBBING_RULE_CONFIGS](https://github.com/DataDog/datadog-serverless-functions/blob/ceda7aad642788ce9b9b2a2bf21668f1f157a6de/aws/logs_monitoring/lambda_function.py#L67-#L76) and the scrubber will take care of "installing" the rules if the environment variable is set, see https://github.com/DataDog/datadog-serverless-functions/blob/ceda7aad642788ce9b9b2a2bf21668f1f157a6de/aws/logs_monitoring/lambda_function.py#L337-#L342

Reply:
👍